lightning/glue/glue.go

```diff
@@ -44,6 +44,7 @@ type SQLExecutor interface {
 	// same underlying connection
 	ExecuteWithLog(ctx context.Context, query string, purpose string, logger log.Logger) error
 	ObtainStringWithLog(ctx context.Context, query string, purpose string, logger log.Logger) (string, error)
+	QueryStringsWithLog(ctx context.Context, query string, purpose string, logger log.Logger) ([][]string, error)
```

Comment:
I'm a little concerned that we will add more QueryTypeWithLog functions here in the future.

Reply:
those other types can still be converted to string 🙃.

we can make it return `[][]interface{}` but so far such generalization isn't needed yet